### ISSUE TITLE
How can I inference single image?

### ISSUE THREAD

**Issue by Ghul-huan** [shahrukhqasim/TIES-2.0#44]

**Comment by sahumson** [comment]
Yes, i have the same doubt..I can only the pickle file loading way..there is no script to inference from single image..Please suggest us how to do it ..thanks
